src/main/target/ANYFCF7/target.h

```diff
@@ -24,6 +24,7 @@
 #define USBD_PRODUCT_STRING "AnyFCF7"
 
 #define USE_DSHOT
+#define USE_ESC_TELEMETRY
```

Comment:
I was wondering about that, and if there was a good place for this. But as I said, just a suggestion to reduce the repetition of common dependency patterns across a lot of targets.